guides/keras_cv/stl10_with_simclr.py

```diff
@@ -0,0 +1,205 @@
+"""
```

Comment:
Please include "contrastive learning" in the filename (more readable when browsing the directories, and good for search engine indexing)